code/game/machinery/doors/airlock_types.dm

```diff
@@ -404,7 +404,7 @@
 	overlays_file = 'icons/obj/doors/airlocks/cult/runed/overlays.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_cult
 	hackProof = TRUE
-	aiControlDisabled = TRUE
+	aiControlDisabled = 1
```

Comment:
this isn't a boolean.

Reply:
🤔 then that variable name is also kind of shitty

Reply:
also shouldnt we be using other defines/flags then instead of just numbers?